10.9-libcxx/stable/main/finkinfo/crypto/gnutls30-shlibs.info

```diff
@@ -88,9 +76,9 @@ InfoTest: <<
   <<
 <<
 Shlibs: <<
-	%p/lib/gnutls30/libgnutls.30.dylib         58.0.0 %n (>= 3.6.13-2)
-	%p/lib/gnutls30/libgnutls-openssl.27.dylib 28.0.0 %n (>= 3.6.13-2)
-	%p/lib/gnutls30/libgnutlsxx.28.dylib       30.0.0 %n (>= 3.6.13-2)
+	%p/lib/gnutls30/libgnutls.30.dylib         65.0.0 %n (>= 3.7.8-1)
+	%p/lib/gnutls30/libgnutls-openssl.27.dylib 28.0.0 %n (>= 3.7.8-1)
+	%p/lib/gnutls30/libgnutlsxx.30.dylib       31.0.0 %n (>= 3.7.8-1)
```

Comment:
Changing the SOVERSION on libgnutlsxx.dylib will need a new package name as we go from 3.6 to 3.7.

Reply:
`libgnutls.30.dylib` -> `gnutls31` ?
Probably better to cherry-pick the first commit then rather than updating everything that depends on gnutls30 (or keep both, perhaps).